.eslintrc.json

```diff
@@ -1,18 +1,16 @@
 {
     "env": {
         "browser": true,
```

Comment:
You work with a node, not a browser

Reply:
Yes. It's my mistake. I fixed it. Thank you.